### PR TITLE
add page for checking ZIP file structure

### PR DESCRIPTION
**Pull request by Adman**
Signed-off-by: Adrian Matejov <ado.matejov@gmail.com>

### CODE REVIEW

**Review by felipenmartins — APPROVED** [review]
It makes sense to me. But I don't know how to test it. :-p 
Nice work!

**Comment by Adman** [comment]
Well, I tested it locally by creating custom html file. Once we merge it it will be deployed so you can have a look :)

image

File: docs/docs/submission_check.md [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+
+                if (errors.length > 0) {
+                    var error_msg = "<ul>";
+                    for (var i = 0; i < errors.length; i++) {
+                        error_msg += "<li>" + errors[i] + "</li>";
+                    }
+                    error_msg += "</ul>";
+                    zip_error.innerHTML = error_msg;
+                } else {
+                   zip_success.innerHTML = "Your ZIP file looks OK!"
```
Comment (mrshu) [inline docs/docs/submission_check.md:38]: @Adman I guess we could make this "green" or something...

Probably next time :)

**Review by mrshu — APPROVED** [review]
Thanks @Adman, let's do it!

File: docs/docs/submission_check.md (hunk 1 above)
Comment (Adman) [reply docs/docs/submission_check.md:38]: It is green ;) check the css

**Pull request merged**